pages/landing.tsx

```diff
@@ -17,12 +17,16 @@ const Landing: FC = () => {
       <h1 className="mb-4">{t('header')}</h1>
       <h2 className="my-14">{t('description')}</h2>
       <div className="flex justify-center flex-wrap text-xl gap-4">
+        <div id="with-esrf">
```

Comment:
I suggest to add `id` prop to `Linkbutton` component and remove the `div` wrapper. Add the `id` to the link `id="link-with-esrf". You might need to update cypress tests.